apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -40,6 +40,46 @@
 // Always include these helper files for dotcom FSE.
 require_once __DIR__ . '/dotcom-fse/helpers.php';
 
+/**
+ * Load Core Site Editor.
+ */
+function load_core_site_editor() {
```

Comment:
What do you think about starting a `site-editor` folder on the same level as `dotcom-fse` and isolating the code of this function there? We'll be adding more stuff for it over time and we probably don't want to keep it all in the top level plugin file where other sub-features are initialized.

Reply:
Absolutely!